pygac/correct_tsm_issue.py

```diff
@@ -0,0 +1,187 @@
+#
+# Copyright (c) Nov 2016, DWD
+#
+# Author(s):
+#
+#   Stephan Finkensieper <stephan.finkensieper@dwd.dwd>
+#   Cornelia Schlundt <cornelia.schlundt@dwd.de>
+#
+# Purpose
+#   NOAA-14, 15 and 16 suffer from temporary scan motor issue
+#   i.e. parts of an orbit contain corrupt data,
+#        which are flagged with fill_values
+#
+
+import os
+import numpy as np
+import scipy.weave as weave
```

Comment:
Agree. Even if scipy is already a dependency, it is hard to debug the inline weave code. Unfortunately there is no 2D version of `np.interp`. But I have been exploring Cython a couple of weeks ago, so I'll try that.